packages/editor-ui/src/modules/workflows.ts

```diff
@@ -0,0 +1,34 @@
+
+import { renameWorkflow } from '@/api/workflows';
+import { ActionContext, GetterTree, Module } from 'vuex';
+import {
+	IRootState,
+	ITag,
+	IWorkflowsState,
+} from '../Interface';
+
+const module: Module<IWorkflowsState, IRootState> = {
+	namespaced: true,
+	state: {},
+	getters: {
+		currentWorkflowTagIds: (state: IWorkflowsState, getters: GetterTree<IWorkflowsState, IRootState>, rootState: IRootState): string[] => {
+			const tags = (rootState.workflow.tags || []) as string[];
```

Comment:
Is there a case where `rootState.workflow.tags` is `undefined` which would require defaulting to an empty array? And in case there is, perhaps:
```suggestion
			const tags = (rootState.workflow.tags ?? []) as string[];
```

Reply:
it will always have a value. maybe these extra checks are unnecessary